src/app/pages/fund/components/fund-account-tile.tsx

```diff
@@ -48,22 +48,22 @@ export function FundAccountTile(props: FundAccountTileProps) {
             height="40px"
             justifyContent="center"
             width="40px"
-            backgroundColor="lightModeInk.1"
+            backgroundColor="oldLightModeInk.1"
```

Comment:
I think we can update this to be `background-primary.value.base`? 

Its using the hex `#FFFFFF`. 

Is that how we assert specific colours now? For example when we want to ignore the theme? I think this needs to ignore the theme

Reply:
I will double check this one. I see I missed a few when search those. Will fix.